ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-danger" role="alert">
+    <p ng-repeat="error in errors">
+        <span class="fa fa-warning"></span> <span class="revision-holder">{{error}}</span>
```

Comment:
Ya, as mentioned above, I think the main reason for the line length discrepancy in the example shown is one SHA is 10 chars, and the other is 11. Fwiw Treeherder's logviewer at one time presented a consistent number of characters regardless of whether the SHA was small, or large. I am not sure if logviewer does that any more, but this was the change at that time.

https://github.com/mozilla/treeherder-ui-deprecated/pull/96/files
`var revision = $scope.artifact.header.revision.substr(0,12);`

In Treeherder itself we applied monospace for other reasons in [this issue](https://github.com/mozilla/treeherder/pull/662).